nbgrader/apps/formgradeapp.py

```diff
@@ -55,6 +55,11 @@ class FormgradeApp(BaseNbGraderApp):
     port = Integer(5000, config=True, help="Port for the server")
     base_directory = Unicode('.', config=True, help="Root server directory")
     directory_format = Unicode('{notebook_id}.ipynb', config=True, help="Format string for the directory structure of the autograded notebooks")
+    start_nbserver = Bool(True, config=True, help="Start a single notebook server that allows submissions to be viewed.")
+
+    @property
+    def notebook_server_exists(self):
```

Comment:
This won't actually get called by the `app` in formgrade.py -- the "app" it's referencing there is the flask app, rather than the FormGradeApp